test/t/base.sh

```diff
@@ -3,30 +3,30 @@
 set -uexo pipefail
 
 compilers=(
-    dmd-2.064
```

Comment:
Hum, this was used to test that the old pattern (`dmd-2.064` as opposed to `dmd-2.064.0`) works, I suppose ?
Could we do it in a way that doesn't run the compiler though ?